assests/JavaScript/logic.js

```diff
@@ -0,0 +1,225 @@
+            <h2 id = "in-which-movie">In which movie is it said:</h2>
+            <h2 id ="question-movie-line">"${movieDataBase[m].line}"</h2>
+            <div class="choice-container"></div>  
+            </div>
+            `)
+        whatMovieOptions()
+        setTimer (playWhatMovie) 
+    } else {
+        endGame()
+    }
+}
+
+//plays who said
+function playWhoSaid (){
+    if (index < randomMovieArray.length) {
+        m = randomMovieArray[index]
+        $('.movie-answer').html(`
+            <div class = "question">
+            <h2 id = "in-which-movie">Who said:</h2>
+            <h2 id ="question-movie-line">"${movieDataBase[m].line}"</h2>
+            <div class="choice-container"></div> 
+            </div>
+            `)
+        whoSaidOptions()
+        setTimer (playWhoSaid) 
+    } else {
+        endGame()
+    }
+}
+
+//sets timer
+function setTimer (game) {
+    let t = 10
+    timer = setInterval (function () {
+        $("#timer").html(t + " seconds")
+        t--
+        if (t < 0) {
+            unanswered++
+            clearInterval(timer)
+            showAnswer('', game)
+        }
+    }, 1000)
+}
+
+//create random choice options/makes sure no duplicates even though speaker in data base more than once
+function whatMovieOptions () {
+    let choiceOptions = []
+    let optionsString = []
+    choiceOptions.push(m)
+    optionsString.push(movieDataBase[m].title)
+    for (i=0; i < 3; i++) {
+        let choice = Math.floor(Math.random()*(moviePool))
+        if (optionsString.indexOf(movieDataBase[choice].title) === -1) {
+            choiceOptions.push(choice)
+            optionsString.push(movieDataBase[choice].title)
+        } else {
+            i--
+        }
+    }
+    shuffle(choiceOptions)
+    for (i = 0; i < 4; i++) {
+        let y = choiceOptions[i]
+        if (y === m) {
+            $('.choice-container').append (`
+                <input type="checkbox" class = "choice-boxes" onclick= "checkAnswer(true, playWhatMovie)">${movieDataBase[y].title}<br>
+                `)
+        } else {
+            $('.choice-container').append (`
+                <input type="checkbox" class = "choice-boxes" onclick= "checkAnswer(false, playWhatMovie)">${movieDataBase[y].title}<br>
+                `)
+        }
+    }
+}              
+
+//create random choice options/makes sure no duplicates even though movie in data base more than once
+function whoSaidOptions () {
+    let choiceOptions = []
+    let optionsString = []
+    choiceOptions.push(m)
+    optionsString.push(movieDataBase[m].speaker)
+    for (i=0; i < 3; i++) {
+        let choice = Math.floor(Math.random()*(moviePool))
+        if (optionsString.indexOf(movieDataBase[choice].speaker) === -1) {
+            choiceOptions.push(choice)
+            optionsString.push(movieDataBase[choice].speaker)
+        } else {
+            i--
+        }
+    }
+    shuffle(choiceOptions)
+    for (i = 0; i < 4; i++) {
```

Comment:
Same here. Where does the number `4` come from? I know that you are showing 4 options for the trivia game. The best way to avoid "magic numbers" is to set a constant at the beginning of your file with a good name. In this case, you would use something like `const numberOfChoices = 4` and then use `numberOfChoices` in your for loop (e.g. `for (...; i < numberOfChoices; ...)`)